src/scene/camera.js

```diff
@@ -532,6 +532,56 @@ class Camera {
         // ortho
         return math.clamp(sphere.radius / this._orthoHeight, 0, 1);
     }
+
+    /**
+     * Returns an array of corners of the frustum of the camera in the local coordinate system of the camera.
+     *
+     * @param {*} near - near distance for the frustum points , defaults to the near clip distance of the camera.
+     * @param {*} far - far distance for the frustum points, defaults to the far clip distance of the camera.
+     * @returns {Array<Vec3>} - An array of corners, using a global storage space.
+     */
+    getFrustumCorners(near, far) {
+
+        near = near || this._nearClip;
+        far = far || this._farClip;
```

Comment:
Maybe use default parameters here?

Reply:
yeah I always feel strange using default parameters that use `this.` - I assume it works? I'd need to test. I will in ES5 and it transpiles to what we have anyways.

Reply:
ok done, works